src/pc/lua/utils/smlua_misc_utils.c

```diff
@@ -426,6 +426,14 @@ const char* get_local_discord_id(void) {
 #endif
 }
 
+const char* get_coopnet_id(s8 localIndex) {
```

Comment:
this function will allow mods to see ip addresses if direct connection is used, which isn't really ideal.